cmd/installer/cli/restore.go

```diff
@@ -631,6 +639,11 @@ func runRestoreEnableAdminConsoleHA(ctx context.Context, flags InstallCmdFlags,
 		addons.WithDomains(getDomains()),
 	)
 
+	kotsadmNamespace, err := runtimeconfig.KotsadmNamespace(ctx, kcli)
```

Comment:
should use the annotation from the backup. all of the restore code in this file should use it